filer/models/abstract.py

```diff
@@ -36,8 +36,8 @@ class BaseImage(File):
     default_alt_text = models.CharField(_('default alt text'), max_length=255, blank=True, null=True)
     default_caption = models.CharField(_('default caption'), max_length=255, blank=True, null=True)
 
-    subject_location = models.CharField(_('subject location'), max_length=64, null=True, blank=True,
-                                        default=None)
+    subject_location = models.CharField(_('subject location'), max_length=64, blank=True,
```

Comment:
Without this change there were a few tests failing because `model_to_dict()` had produced data with `subject_location=None` but form validation did not allow to save it (as expected). So, if there are no really good reasons to keep `Null=True` for a text field here, it will be removed.